### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by oorjadandotiya**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -267,3 +267,8 @@
   title: Here I come Coding In
   img: haider2.jpg
   author_name: Haider Shafaq
+  
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title: Technoorja Learns To Code
+  img: Oorja_modified.jpg
```
Comment (nikit19) [inline _data/blogs.yml:273]: Looks good but upload this image

**Review by nikit19 — CHANGES_REQUESTED** [review]

File: _data/blogs.yml (hunk 1 above)
Comment (oorjadandotiya) [reply _data/blogs.yml:273]: I have uploaded that image

Comment (nikit19) [reply _data/blogs.yml:273]: Where?

Comment (oorjadandotiya) [reply _data/blogs.yml:273]: I have uploaded that image.

Comment (oorjadandotiya) [reply _data/blogs.yml:273]: In the img/blog  folder

Comment (codedsun) [reply _data/blogs.yml:273]: @oorjadandotiya The image is not uploaded in this commit!

**Review by codedsun — CHANGES_REQUESTED** [review]

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
@oorjadandotiya Please refrain from opening so many same PRs, without any improvement :+1:

File: _data/blogs.yml (hunk 1 above)
Comment (oorjadandotiya) [reply _data/blogs.yml:273]: @codedsun 
Just re-uploaded the image to this commit - https://github.com/oorjadandotiya/gci17.fossasia.org/commits/gh-pages

Comment (oorjadandotiya) [reply _data/blogs.yml:273]: Please confirm if this has completed successfully.

**Review by oorjadandotiya — COMMENTED** [review]
Please review the changes.

**Comment by anshumanv** [comment]
Things are still the same, please upload an image in your fork.

**Comment by oorjadandotiya** [comment]
I already did that. please refer to the screenshot below:
[image: Inline images 1]

File: _data/blogs.yml (hunk 1 above)
Comment (oorjadandotiya) [reply _data/blogs.yml:273]: Just uploaded the image. PFB the link. 
https://github.com/oorjadandotiya/gci17.fossasia.org/blob/gh-pages/img/blog/Oorja1.png
Hope this time it works.

**Comment by oorjadandotiya** [comment]
Dear Mentor,
https://github.com/oorjadandotiya/gci17.fossasia.org/blob/gh-pages/img/blog/Oorja1.png
I have added this image.

Please find attached the screenshot of the image uploaded.

**Review by ParthS007 — COMMENTED** [review]
@oorjadandotiya Please go through README.md on how to add a blog and make a new PR.You can ask on FOSSASIA gitter channel if you face any difficulty. 👍

**Pull request closed without merging**